fetch.bs

```diff
@@ -4363,10 +4363,30 @@ runs the associated steps:
   run these substeps:
 
   <ol>
-   <li><p>Parse <var>bytes</var>, using the value of the `<code>boundary</code>` parameter from
-   <var>MIME type</var> and <a>utf-8</a> as encoding, per the rules set forth in
-   <cite>Returning Values from Forms: multipart/form-data</cite>.
-   [[!RFC7578]]
+   <li>
+    <p>Parse <var>bytes</var>, using the value of the `<code>boundary</code>` parameter from
+    <var>MIME type</var> and <a>utf-8</a> as encoding, per the rules set forth in
+    <cite>Returning Values from Forms: multipart/form-data</cite>.
+    [[!RFC7578]]</p>
+
+    <p>Each part whose `<code>Content-Disposition</code>` header contains
+    a `<code>filename</code>` parameter must be parsed into an <a>entry</a>
+    whose value is a {{File}} whose content is the content of the part. The {{File/name}} attribute
+    of the {{File}} must have the value of the `<code>filename</code>` parameter of the part.
+    The {{Blob/type}} attribute of the {{File}} must have the value of
+    the `<code>Content-Type</code>` header of the part if the part has such header or
+    `<code>text/plain</code>` (the default defined by [[!RFC7578]] section 4.4) otherwise.</p>
+
+    <p>Each part whose `<code>Content-Disposition</code>` header does not contain
+    a `<code>filename</code>` parameter must be parsed into an <a>entry</a>
+    whose value is a {{USVString}} whose content is the <a>utf-8</a> decoded content of the part.
+    This is done regardless of the presence or the value of a `<code>Content-Type</code>` header
+    and regardless of the presence or the value of a `<code>charset</code>` parameter.</p>
+
+    <p>A part whose `<code>Content-Disposition</code>` header contains a `<code>name</code>`
```

Comment:
I think this would be clearer as a note (class=note) since it's just a statement of fact.